src/amuse/rfi/channel.py

```diff
@@ -44,9 +44,11 @@
 from . import async_request
```

Comment:
This file is such a disaster. But at least it's now a nicely formatted disaster. Great!